core/src/main/java/com/rudderstack/android/sdk/core/RudderContext.java

```diff
@@ -44,6 +44,10 @@ public class RudderContext {
     private RudderNetwork networkInfo;
     @SerializedName("timezone")
     private String timezone;
+    @Nullable @SerializedName("sessionId")
+    private String sessionId = null;
+    @Nullable @SerializedName("sessionStart")
+    private Boolean sessionStart = null;
```

Comment:
`sessionStart` will be added under context for the first event only. So, creating an object for a single key-value pair is unnecessary. And it will be easy to parse on the Transformer side.

Reply:
My idea here is that, we might need to add few more properties as we add support for more destinations